ApplicationLibCode/ProjectDataModel/RimProject.cpp

```diff
@@ -614,6 +614,8 @@ std::vector<RimCase*> RimProject::allGridCases() const
 
             for ( RimEclipseCaseEnsemble* ensemble : analysisModels->caseEnsembles() )
             {
+                if ( !ensemble ) continue;
```

Comment:
Why could there be nullptrs in the ensemble?

Reply:
Seen in destructor of `PdmProject` as part of clean up. Appears that the destructor of ensembles has been run before this code is executed. I will postpone this fix until I find a reliable way to reproduce this issue.